lib/engine-addon.js

```diff
@@ -362,6 +362,18 @@ module.exports = {
       }).filter(Boolean);
     };
 
+    /**
+     * Determines if ember-engines should use the compact-reexports Babel plugin.
+     */
+    options.shouldCompactReExports = function() {
+      var VersionChecker = require('ember-cli-version-checker');
+      var checker = new VersionChecker(this);
+
+      return checker.for('ember-cli', 'npm').satisfies('>=2.13.0') &&
+        checker.for('ember-cli-babel', 'npm').satisfies('>=6.0.0') &&
```

Comment:
I got lost down the rabbit hole in the version checker, I ended up completely refactoring it to use ES6 before working on the thing I actually needed to do.

> in the interim would the best solution be to findAddonByName check its package directly?

Yes, `this.addons.find(addon => addon.name === 'ember-cli-babel').pkg.version` or something?